components/FlexibleForms/FlexibleFields.test.tsx

```diff
@@ -0,0 +1,86 @@
+import FlexibleFields from './FlexibleFields';
+import { Formik } from 'formik';
+import { render, screen } from '@testing-library/react';
+
+const mockSubmit = jest.fn();
+
+describe('TextField', () => {
```

Comment:
We're only testing TextField renders correctly but there are quite a few different field options, do we need to test those render as expected.

Reply:
each of those fields is individually tested - doing it again here seems like unnecessary duplication?

Reply:
What I am looking for is that under certain conditions we choose to render the correct "Field" such as a TextField or a RadioButton etc.

Reply:
it'll be one `it()` for every field, probably. a bit verbose. that okay?

Reply:
done